src/app/games/trexo/TrexoTutorial.ts

```diff
@@ -0,0 +1,74 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { Coord } from 'src/app/jscaip/Coord';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { TrexoMove } from './TrexoMove';
+import { TrexoRules } from './TrexoRules';
+import { TrexoState } from './TrexoState';
+
+const initialState: TrexoState = TrexoState.getInitialState();
+const moves: TrexoMove[] = [
+    TrexoMove.from(new Coord(4, 4), new Coord(4, 3)).get(),
+    TrexoMove.from(new Coord(6, 4), new Coord(5, 4)).get(),
+    TrexoMove.from(new Coord(5, 5), new Coord(5, 6)).get(),
+    TrexoMove.from(new Coord(4, 4), new Coord(5, 4)).get(),
+    TrexoMove.from(new Coord(7, 3), new Coord(8, 3)).get(),
+    TrexoMove.from(new Coord(3, 1), new Coord(3, 2)).get(),
+    TrexoMove.from(new Coord(7, 5), new Coord(6, 5)).get(),
+];
+let previousState: TrexoState = initialState;
+const movesAndStates: { move: TrexoMove, state: TrexoState }[] = [];
+for (const move of moves) {
+    const state: TrexoState = TrexoRules.get().applyLegalMove(move, previousState);
+    movesAndStates.push({ move, state });
+    previousState = state;
+}
+const firstMove: TrexoMove = TrexoMove.from(new Coord(5, 5), new Coord(5, 4)).get();
+const stateAtTurnOne: TrexoState = TrexoRules.get().applyLegalMove(firstMove, initialState);
+const secondMove: TrexoMove = TrexoMove.from(new Coord(5, 7), new Coord(5, 6)).get();
+const stateAtTurnTwo: TrexoState = TrexoRules.get().applyLegalMove(secondMove, stateAtTurnOne);
+
+export class TrexoTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        // 1. but du jeu et plateau
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`At Trexo, the goal of the game is to align 5 piece of your color in an horizontal, vertical or diagonal line. But player must put a tile on the board, and that tile is constituted of two pieces, one from each players!`,
```

Comment:
an horizontal -> a horizontal (h consonne)
Je reformulerai la deuxième phrase ainsi : "However, the players play with tiles that are constituted of two pieces, one from each player"

Reply:
+ 5 piece -> 5 pieces